transforms/universal/pdf2md/python/Makefile

```diff
@@ -0,0 +1,67 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+
+# $(REPOROOT)/.make.versions file contains the versions
+
+TRANSFORM_RUNTIME=python
```

Comment:
TRANSFORM_RUNTIME should not be need now that you are in a `python` directory.

Reply:
done